bentoml/_internal/frameworks/catboost.py

```diff
@@ -95,9 +89,7 @@ def load(
     tag: t.Union[str, Tag],
     model_params: t.Optional[t.Dict[str, t.Union[str, int]]] = None,
     model_store: "ModelStore" = Provide[BentoMLContainer.model_store],
```

Comment:
I saw that we removed `model_store` inject from `load_runner`, should we also do that for `save` and `load`?

I think it makes more sense to just use public API under `bentoml.models`  in our framework modules - I don't think allowing user to customize model_store for each `save` or `load` call is a valid use case, given that 1.0 will bring in a pretty powerful export/import feature that @sauyon is working on.

Reply:
Let's do that in another PR